bench-exchange/src/bench.rs

```diff
@@ -1105,4 +1108,29 @@ mod tests {
 
         do_bench_exchange(clients, config);
     }
+
+    #[test]
+    fn test_exchange_bank_client() {
+        solana_logger::setup();
+        let (mut genesis_block, identity) = GenesisBlock::new(100_000_000_000_000);
+        genesis_block
+            .native_instruction_processors
+            .extend_from_slice(&[(
```

Comment:
Use `bank.add_instruction_processor()` so that you don't need to manually `cargo build` and you get the debug symbols (for better stack traces and flame graphs).